core/src/main/java/org/apache/carbondata/core/indexstore/ExtendedBlocklet.java

```diff
@@ -221,7 +221,7 @@ public void deserializeFields(DataInput in, String[] locations, String tablePath
       indexUniqueId = in.readUTF();
     }
     String filePath = getPath();
-    if (filePath.startsWith(File.separator)) {
+    if (!FileFactory.isFileExist(filePath)) {
```

Comment:
Please revert this change to do a rpc call. Use FileFactory.getupdatedFilePath instead as discussed

Reply:
As discussed, using FileExists only for local file path.